pipelines/B2/B2.R

```diff
@@ -0,0 +1,71 @@
+# -------------------------------------------------------------------------
+# Step: Get the data
+# input: data/const/statbot_input_data.csv
+# output: ds as dataset class
+# -------------------------------------------------------------------------
+
+ds <- statbotData::create_dataset("B2")
+ds <- statbotData::download_data(ds)
+ds_boys <- ds$data %>%
+  janitor::clean_names() %>%
+  dplyr::mutate(gender = "boy")
+ds_boys
+# get additional datasets for girls
+ds$px_id <- "px-x-0104050000_102"
+ds$download_url <- "https://www.pxweb.bfs.admin.ch/DownloadFile.aspx?file=px-x-0104050000_102"
+ds <- statbotData::download_data(ds)
+
+ds_girls <- ds$data %>%
+  janitor::clean_names() %>%
+  dplyr::mutate(gender = "girl")
+ds_girls
+
+ds$data <- ds_boys %>%
+  dplyr::bind_rows(ds_girls)
+ds$data
+
+# -------------------------------------------------------------------------
+# Step: Clean the data
+#   input: ds$data
+#.  output: ds$cleaned_data: separate data by units
+# -------------------------------------------------------------------------
+
+ds$cleaned_data <- ds$data %>%
+  tidyr::pivot_wider(
+    names_from = unit_of_measure,
+    values_from = data_person
+  ) %>%
+  janitor::clean_names(
+  ) %>%
+  dplyr::rename(
+    amount = number,
+    canton = language_region_canton
```

Comment:
**suggestion(non-blocking):** use a more general column name such as `region` or `location` instead of canton, as it also includes Switzerland.